src/dropin-model.js

```diff
@@ -219,29 +249,32 @@ DropinModel.prototype.reportAppSwitchError = function (sheetId, error) {
   };
 };
 
-DropinModel.prototype.asyncDependencyStarting = function () {
-  this.dependenciesInitializing++;
+DropinModel.prototype.hasAtLeastOneAvailablePaymentOption = function () {
+  var self = this;
+  var i;
+
+  console.log(this.supportedPaymentOptions);
```

Comment:
These console logs look like they can be removed.

Reply:
Yep, nice catch.